airbyte-webapp/src/components/icons/CrossIcon.tsx

```diff
@@ -0,0 +1,12 @@
+interface CrossIconProps {
```

Comment:
Is there a specific reasons, we're not going with the font awesome pencil and cross icon here?

Reply:
We should discuss this with @Upmitt, but there was a a specific reason that the fontawesome should be replaced with this....